fns-hl7-pipeline/fn-redactor/src/main/kotlin/gov/cdc/dex/hl7/Function.kt

```diff
@@ -69,15 +66,17 @@ class Function {
                     routeElement.asString
                 }
 
-                logger.info("DEX:: Received and Processing messageUUID: $messageUUID, filePath: $filePath")
+				logger.info("DEX:: Received and Processing messageUUID: $messageUUID, filePath: $filePath")
 
                 val report = helper.getRedactedReport(hl7Content, messageType, route)
+
                 if(report != null) {
                     val rReport = RedactorReport(report._2())
                     val configFileName : List<String> = listOf(helper.getConfigFileName(messageType, route))
                     val processMD = RedactorProcessMetadata(rReport.status, report = rReport, eventHubMD[msgIndex], configFileName)
                     processMD.startProcessTime = startTime
                     processMD.endProcessTime = Date().toIsoString()
+                    context.logger.info("Process MD: ${processMD} ")
```

Comment:
remove context, it should just be `logger.info`

Reply:
fixed.  Commit Id: 8dafe49